[CI] XpackWithMultipleProjectsClientYamlTestSuiteIT test {yaml=esql/60_usage/Basic ESQL usage output (telemetry) snapshot version} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #10853 / windows-2025_checkpart2_platform-support-windows](https://gradle-enterprise.elastic.co/s/7d3rxepuktn2w)
- [elasticsearch-pull-request #95333 / part-2](https://gradle-enterprise.elastic.co/s/wd5ff7uvxdbpy)

**Reproduction Line:**
```
gradlew ":x-pack:qa:multi-project:xpack-rest-tests-with-multiple-projects:yamlRestTest" --tests "org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT.test {yaml=esql/60_usage/Basic ESQL usage output (telemetry) snapshot version}" -Dtests.seed=8124B1A1D768D267 -Dtests.locale=kln-Latn-KE -Dtests.timezone=Asia/Jerusalem -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Desql%2F60_usage%2FBasic%20ESQL%20usage%20output%20(telemetry)%20snapshot%20version%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [esql/60_usage:204]: field [esql.functions.min] is not greater than [$functions_avg_over_time]
Expected: a value greater than <1>
     but: <1> was equal to <1>
```

**Issue Reasons:**
- [main] 2 failures in test test {yaml=esql/60_usage/Basic ESQL usage output (telemetry) snapshot version} (2.3% fail rate in 86 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -411,9 +411,6 @@ tests:
 - class: org.elasticsearch.xpack.search.CrossClusterAsyncSearchIT
   method: testCancelViaExpirationOnRemoteResultsWithMinimizeRoundtrips
   issue: https://github.com/elastic/elasticsearch/issues/127302
-- class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlClientYamlIT
-  method: test {p0=esql/60_usage/*}
-  issue: https://github.com/elastic/elasticsearch/issues/133461
 - class: org.elasticsearch.xpack.esql.action.CrossClusterAsyncQueryStopIT
   method: testStopQueryLocal
   issue: https://github.com/elastic/elasticsearch/issues/133481
@@ -555,9 +552,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.inference.textembedding.TextEmbeddingOperatorTests
   method: testSimpleCircuitBreaking
   issue: https://github.com/elastic/elasticsearch/issues/135569
-- class: org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT
-  method: test {yaml=esql/60_usage/Basic ESQL usage output (telemetry) snapshot version}
-  issue: https://github.com/elastic/elasticsearch/issues/135579
 - class: org.elasticsearch.xpack.esql.heap_attack.HeapAttackIT
   method: testAggTooManyMvLongs
   issue: https://github.com/elastic/elasticsearch/issues/135585
@@ -627,9 +621,6 @@ tests:
 - class: org.elasticsearch.xpack.security.CoreWithSecurityClientYamlTestSuiteIT
   method: test {yaml=indices.get_sample/10_basic/Test get sample for index with no sample config}
   issue: https://github.com/elastic/elasticsearch/issues/135993
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version}
-  issue: https://github.com/elastic/elasticsearch/issues/136000
 - class: org.elasticsearch.xpack.ml.integration.RegressionIT
   method: testAliasFields
   issue: https://github.com/elastic/elasticsearch/issues/135996

--- a/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml
+++ b/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml
@@ -350,7 +350,7 @@ setup:
   - gt: { esql.queries._all.total: $all_total_counter }
   - match: { esql.queries._all.failed: $all_failed_counter }
   - gt: { esql.functions.min: $functions_min }
-  - gt: { esql.functions.min: $functions_avg_over_time }
+  - gt: { esql.functions.avg_over_time: $functions_avg_over_time }
 
   # There's one of these per function but that's a ton of things to check. So we just spot check that a few exist.
   - not_exists: esql.functions.delay

